10X_v2/analysis/analysis.json

```diff
@@ -0,0 +1,80 @@
+{
+   "input_bundles" : [ "bluebox://23bd7eb0-23a3-4898-b043-f7e982de281f/v1" ],
+   "inputs" : [
+      {
+        "name": "left_sample",
+        "checksum" : "d0f7d08f1980f7980f",
+        "value" : "bluebox://23bd7eb0-23a3-4898-b043-f7e982de281f/v1/12bd7eb0-23a3-4898-b043-f7e982de281g/v1#10x_R1.fastq.gz"
+      },
+      {
+         "name" : "right_sample",
+         "checksum" : "d0f7d18f7980f7980f",
+         "value" : "bluebox://23bd7eb0-23a3-4898-b043-f7e982de281f/v1/01bd7eb0-23a3-4898-b043-f7e982de281g/v1#10x_R2.fastq.gz"
+      },
+      {
+         "name" : "index_sample",
+         "checksum" : "d0f7d18f7980f7980f",
+         "value" : "bluebox://23bd7eb0-23a3-4898-b043-f7e982de281f/v1/01bd7eb0-23a3-4898-b043-f7e982de281g/v1#10x_I1.fastq.gz"
+      },
+      {
+         "name" : "expected_cell_count",
+         "value" : "1000"
+      }
+   ],
+   "timestamp_start_utc" : "2017-05-18T03:44:23-5:00",
+   "timestamp_stop_utc" : "2017-05-18T014:44:23-5:00",
+   "computational_method" : "https://www.dockstore.org:8443/api/ga4gh/v1/tools/quay.io%2Fhca%2Fdockstore-hca-uploader/",
```

Comment:
Should we call this "pipeline" instead? I found this confusing the first time I saw it, because I think of a method being a single task in a WDL, like running STAR or bwa.